models/user/user.go

```diff
@@ -1233,7 +1233,10 @@ func GetUserByOpenID(uri string) (*User, error) {
 // GetAdminUser returns the first administrator
 func GetAdminUser() (*User, error) {
 	var admin User
-	has, err := db.GetEngine(db.DefaultContext).Where("is_admin=?", true).Get(&admin)
+	has, err := db.GetEngine(db.DefaultContext).
+		Where("is_admin=?", true).
+		OrderBy("id asc"). // Reliably get the admin with the lowest ID.
```

Comment:
```suggestion
		Asc("id"). // Reliably get the admin with the lowest ID.
```
Looks like we lost the review change. 
Thanks again for the PR!